src/app/app-routing.module.ts

```diff
@@ -1,10 +1,28 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { AuthGuard } from './core/guards/auth.guard';
 
-const routes: Routes = [];
+const routes: Routes = [
+  // { path: 'login', component: AuthComponent },
+  {
+    path: 'login',
+    loadChildren: () => import('./login/login.module').then(m => m.LoginModule),
+  },
+  {
+    path: 'admin',
+    canActivate: [AuthGuard],
+    loadChildren: () => import('./admin/admin.module').then(m => m.AdminModule),
+  },
+  {
+    path: '',
+    loadChildren: () => import('./user/user.module').then(m => m.UserModule),
+  },
+  { path: 'login', loadChildren: () => import('./login/login.module').then(m => m.LoginModule) },
```

Comment:
please follow writing pattern as for prev items

chose either writing everything in one line, either separating each item on a new line

Reply:
fixed